### ISSUE TITLE
FINOS wiki and website cleanup post new governance publication 

### ISSUE THREAD

**Issue by mindthegab** [finos/community#48]
Once (or contextually with) we have completed with #45 and it's successfully merged here, we should look at https://finosfoundation.atlassian.net/wiki/spaces/FINOS and finos.org/governance and:

- [ ] De-duplicate information which is now in this repository (e.g. editable policies)
- [ ] Update website links to point to this information
- [ ] Remove any program related remaining information
- [ ] Create a central entry point for contributors, once that consolidates at least https://www.finos.org/get-involved and https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/83034172/Contribute
- [ ] TBC

**Comment by brooklynrob** [comment]
Along the lines of https://finosfoundation.atlassian.net/browse/HELP-471, I would suggest that on https://www.finos.org/governance there be a link to the community/project governance (i.e., this site)

As further context/background there is not presently a link the existing FINOS project governance (e.g., the community handbook, https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80642059/Community+Handbook) on https://www.finos.org/governance.

tl;dr --> I think https://www.finos.org/governance should include a link to our projects' governance policies.

**Comment by mindthegab** [comment]
@toshaellison you mentioned you had some material here re inventory of links to the wiki? or something related to a website audit? can't remember but if you have anything to help get the work started please post it here. thx!

**Comment by toshaellison** [comment]
The CLA Bot email directing new contributors that they need a CLA points to https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Contribution+Compliance+Requirements#LegalRequirements-ContributorLicenseAgreement

I've updated that page to point to the new CLAs.

**Comment by toshaellison** [comment]
This is a little long but hopefully captures most of the places that changes still need to be made or a decision made about whether they should change now. (For obvious ones I've just made the changes.) And this relates primarily to governance.

**Active Participation**
Question: Do we need to change this or remove references to it?
Current finos.org references:
- finos.org/governance (https://www.finos.org/hubfs/FINOS/governance/FINOS%20Active%20Participation%20Policy%202018-12-12.pdf?hsLang=en-us)
- finos.org/faq|"Marketing, Branding, Trademarks"|"Can I advertise my participation in a FINOS project?"
- finos.org/governance|Active Participation Policy
- finos.org/faq|"Meetings"|"Is attendance to FINOS working group meetings open to the public?"|"Foundation's Active Participation Policy"

**finos.org/FAQ** 
Questions/To Be Updated: 
- Where should should finos.org/faq|"Can I advertise my participation in a FINOS project?"|Trademark Guidelines" link to (current is https://www.finos.org/hubfs/FINOS/governance/FINOS%20Trademark%20Guidelines%202018-12-12.pdf?hsLang=en-us)
- Update question "Do the chairs of archived working groups remain as PMC members?"
- There is a link from faq|"Open Source Projects"|"What is a good granularity for a project" to    https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530523/Adding+a+New+Repository+to+your+Project. Is it sill appropriate?
- Reference to PMCs to be fixed in faq/"Meetings"|May non PMC members attend and participate in PMC meetings?"


**finos.org links to Wiki Community Handbook page**
The below on finos.org link to https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/83034172/Contribute
- finos.org/Resources|Community Handbook
- engage-with-our-community|"Lead"|"Learn how our community is governed"
- faq|"Meetings"|How are meetings prepared? Who keeps..."
- faq|"Meetings"|What level of detail is expected in minutes..."
- faq|"Meetings"|How does attribution work in minutes..."


**finos.org links to Wiki Contribute page**
The below on finos.org link to https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/83034172/Contribute
- Projects|contribute
- engage-with-our-community|"Contribute"|"Contribute a new project"
- engage-with-our-community|"Lead"|"Contribute a new project"
- get-involved|"Experienced contributor"|"Contribute"
- faq|"Legal and Licensing"|"Who owns my project after it's been contributed to FINOS"|"contribution page"
- faq|"Open Source Projects"|"Are there any restrictions on how many projects I can host with the Foundation?"


**finos.org links to Wiki Project + Lifecycle page**
The below on finos.org links to https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530756/Project+Lifecycle
- engage-with-our-community|"Consume"|"Project Maturity"
- faq|"Open Source Projects"|"What is good granularity for a project?"


**finos.org links to Wiki Contribution+Compliance+Requirements page**
https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Contribution+Compliance+Requirements
- engage-with-our-community|"Contribute"|"Contributor Agreement"
- get-involved|"Contributor License Agreement"|"FINOS CLA requirements can be found on the FINOS Wiki"
- get-involved|"Experienced contributor"|"contributor license agreement"
- faq|"Who owns my project after it's been contributed to FINOS"|"Foundation's CLA"


**finos.org links to Wiki Learn page**
The below on finos.org links to https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/83197975/Learn
- get-involved|"New to Open Source"|"Learning more"


**Wiki pages to be updated (not exhaustive)**
- https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530523/Adding+a+New+Repository+to+your+Project (reference to PMC, catalogue)
- https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/83197975/Learn (reference to Programs and link to Governance)

**Comment by mindthegab** [comment]
1st pass, tracking here a log of what's done, feel free to review @toshaellison @brooklynrob @mcleo-d @maoo  (not finished but at a consistent point - I have changed no links, so all should still be accessible):

### Website 
- **Active Participation** Question: Do we need to change this or remove references to it? Answer: Active Participation now matters only in the Standard process and for our own marketing decision, as for software projects we have a clear definition of maintainers / roster. So i've removed it from finos.org/governance, but remains linked throughout until we close the loop.
-  [FAQ](finos.org/FAQ): Adjusted verbiage all around and removed obsolete questions
-  [Governance](finos.org/governance): Reorganized content and linked to the Community Governance in GIthub 

### Wiki 

- Archived (link remains the same), added custom deprecation notices to each pages (pointing to new content) and moved content if still valid on the following pages:
  - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530351/Decision+Making
  - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530783/Community+Governance
  - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/81166347/Foundation+Structure 
  - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/92963365/Program-Level+Processes and sub pages
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/92996354/Providing+Steering
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/804028435/Program+Roadmaps
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/94143035/Approving+Lifecycle+Transitions
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530355/Adding+a+New+Project+Team+Member
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/92963520/Approving+new+Projects+and+Working+Groups
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/92995960/Program+Onboarding
  - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80315025/Working+Group-Level+Processes and sub pages
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/154468392/Working+Group+Onboarding
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/95846592/Adding+a+New+Working+Group+Participant
     - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/92930865/Requesting+Steering+Working+Group
- Archived https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/1191903251/Running+a+Standards+Project in favor of https://github.com/copiesofcopies/community/tree/governance/governance/Standards-Projects
- Archived https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/1124302852/Ranked-Choice+Voting+for+Board+Elections in favor of https://github.com/copiesofcopies/community/blob/governance/governance/Board-Election-Procedure.md
- Archived https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/92996572/Running+Good+Meetings in favor of https://github.com/finos/community/blob/master/governance/Meeting-Procedures.md
- Updated project pages to remove program / PMC references
    - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530357/Requesting+Steering+Project
    - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530523/Adding+a+New+Repository+to+your+Project
    - https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530353/Accepting+Contributions
- Archived ODP pages now deprecated in favor of odp.finos.org
  - https://finosfoundation.atlassian.net/wiki/spaces/FDX/pages/94142628/Collaboration+Services
  - https://finosfoundation.atlassian.net/wiki/spaces/FDX/pages/1057259529/ARCHIVED+Groups.io
- **Drastically** simplified [community handbook](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80642059/Community+Handbook)

This is the first pass...
